light_model:
  x_min: 0.0
  x_good: 1.0

quality:
  score_threshold: 0.8

trace:
  buffer_size: 30

move:
  nudge_duration_sec: 30.0
